app/views/admin/events/_datatable_row.haml

```diff
@@ -24,11 +24,15 @@
         Unknown submitter
 
     %td
-      - event.speakers_ordered.each do |speaker|
-        .speaker
-          = link_to speaker.name, admin_user_path(speaker)
-          - unless speaker.registrations.for_conference(event.conference)
-            %span.label.label-danger Unregistered
+      - if event.speakers.any?
+        - event.speakers_ordered.each do |speaker|
+          .speaker
+            = link_to speaker.name, admin_user_path(speaker)
+            - unless speaker.registrations.for_conference(event.conference)
+              %span.label.label-danger Unregistered
+      - else
+        .small
+          No Speaker
```

Comment:
We don't want to show the users _No speaker_. This is good for the admin (as he may want to know this information), but not for the rest of the users. I think that not showing anything in this case is fine. :wink:

Reply:
I thought this view is for admin, since it is in the admin folder.